### PR TITLE
[BE] Show simple table of statusewise count

### PR DESCRIPTION
**Pull request by Tusharmahajan12**

### CODE REVIEW

**Comment by manojLondhe** [comment]
@Tusharmahajan12 resolve conclfits and format file

**Review by manojLondhe — CHANGES_REQUESTED** [review]

File: src/src/beneficiaries/beneficiaries.controller.ts [hunk 1]
```diff
@@ -19,31 +19,36 @@ import { RegisterBeneficiaryDto } from './dto/register-beneficiary.dto';
...
-  @Req() req:any, @Res() response: Response) {
-    return this.beneficiariesService.findAll(request,req,response);
+  findAll(
+    @Body() request: Record<string, any>,
+    @Req() req: any,
+    @Res() response: Response,
+  ) {
+    return this.beneficiariesService.findAll(request, req, response);
+  }
+  @Get('/statusCount')
```
[the remark this answers is not in the document]
Comment (Tusharmahajan12) [reply src/src/beneficiaries/beneficiaries.controller.ts]: resolved

File: src/src/beneficiaries/beneficiaries.controller.ts [hunk 2]
```diff
@@ -19,31 +19,36 @@ import { RegisterBeneficiaryDto } from './dto/register-beneficiary.dto';
...
+  findAllStatusCount(@Req() request: any, @Res() response: Response) {
```
[the remark this answers is not in the document]
Comment (Tusharmahajan12) [reply src/src/beneficiaries/beneficiaries.controller.ts]: resolved

File: src/src/beneficiaries/beneficiaries.service.ts [hunk 3]
```diff
@@ -8,56 +13,96 @@ import { HasuraService as HasuraServiceFromServices } from '../services/hasura/h
...
+      'registered_in_camp',
+      'pragati_syc',
+      'rejected',
+      'dropout',
+    ];
+    let qury = `query MyQuery {
+      ${status.map(
+        (
+          item,
+        ) => `${item}:beneficiaries_aggregate(where: {status: {_eq: "${item}"}}) {
```
Comment (manojLondhe) [inline src/src/beneficiaries/beneficiaries.service.ts:64]: add where condition where beneficiary_id = logged in prerak's id

**Review by manojLondhe — CHANGES_REQUESTED** [review]

File: src/src/beneficiaries/beneficiaries.service.ts (hunk 3 above)
Comment (Tusharmahajan12) [reply src/src/beneficiaries/beneficiaries.service.ts]: resolved sir

**Pull request merged**